deployments/1.14.x/addons.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: AddonsDeployment
+metadata:
+  name: defaultaddons-1.14.x
```

I would rename all of them to `default-addons-deployment`. They also would be going into the `kubeaddons` namespace.

---

The file name should probably match the name :)

---

Would you add the Kubernetes version to the name or would you prefer all the AddonsDeployment to be called `default-addons-deployment`?

---

Would it be more appropriate to name this more precisely according to its intended application?

e.g.:

```suggestion
  name: default-ksphere-addons-1.14.x
```
Or something like that? Just wondering what you think, not a blocker.
```